src/effects/native/balanceeffect.cpp

```diff
@@ -0,0 +1,182 @@
+#include "balanceeffect.h"
+
+#include "util/defs.h"
+
+namespace {
+    const double kMaxCornerHz = 500;
+    const double kMinCornerHz = 16;
+    const unsigned int kStartupSamplerate = 44100;
+} // anonymous namespace
+
+// static
+QString BalanceEffect::getId() {
+    return "org.mixxx.effects.balance";
+}
+
+// static
+EffectManifest BalanceEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Balance"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr("Adjust the left/right balance and Stereo width"));
+    manifest.setEffectRampsFromDry(true);
+
+    EffectManifestParameter* balance = manifest.addParameter();
+    balance->setId("balance");
+    balance->setName(QObject::tr("Balance"));
+    balance->setDescription(QObject::tr("Adjust balance between left and right channels"));
+    balance->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
+    balance->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    balance->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    balance->setDefaultLinkType(EffectManifestParameter::LinkType::LINKED);
+    balance->setMinimum(-1.0);
+    balance->setMaximum(+1.0);
+    balance->setDefault(0.0);
+
+    EffectManifestParameter* midSide = manifest.addParameter();
+    midSide->setId("midSide");
+    midSide->setName(QObject::tr("Mid/Side"));
+    midSide->setDescription(QObject::tr("Adjust balance between mid and side. Fully mid is Mono."));
+    midSide->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
+    midSide->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    midSide->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    midSide->setDefaultLinkType(EffectManifestParameter::LinkType::NONE);
+    midSide->setMinimum(-1.0);
+    midSide->setMaximum(+1.0);
+    midSide->setDefault(0.0);
+
+    EffectManifestParameter* midLowPass = manifest.addParameter();
+    midLowPass->setId("bypassFreq");
+    midLowPass->setName(QObject::tr("Bypass Freq."));
```

Comment:
That is what "Lorem ipsu" does. At least it is much better than a pure character count.
The capitalized text in Deere works not that well, by the way. If the text requires lower and higher case letters to be recognizable. IMHO it is also harder to read.